### PR TITLE
Use classloader for version

### PR DESCRIPTION
**Pull request by andrewlmurray**

### CODE REVIEW

**Comment by andrewlmurray** [comment]
Release song and dance!

**Comment by markschaake** [comment]
Assigning to @jkinkead since he has questions in https://github.com/allenai/common/pull/175

**Comment by andrewlmurray** [comment]
Sounds good! @jkinkead I responded to you in #175

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea/
```
Comment (jkinkead) [inline .gitignore]: This should go in your `~/.gitignore`.

**Comment by jkinkead** [comment]
I'm slightly concerned that this will introduce bugs into downstream dependencies, since it changes how `.conf` files are found - but honestly, that's probably already classpath-order dependent, and we don't make an effort to sort that in a sensible way.

I'm also worried that this is a workaround for another bug, but it ought to be harmless either way.

**Pull request merged**